FindPEGTL.cmake

```diff
@@ -21,6 +21,32 @@
 #  find_package(PEGTL)
 #  include_directories(${PEGTL_INCLUDE_DIRS})
 
+function(_PEGTL_GET_VERSION _OUT_major _OUT_minor _OUT_micro _metisversion_hdr)
+    file(STRINGS ${_metisversion_hdr} _contents REGEX "#define PEGTL_VER_[A-Z]+[ \t]+")
+    if(_contents)
+        string(REGEX REPLACE ".*#define TAOCPP_PEGTL_VERSION_MAJOR[ \t]+([0-9]+).*" "\\1" ${_OUT_major} "${_contents}")
```

Comment:
There may have been a visual artifact  that made it look like there was a one space delta and this may not actually be an issue now that I look at the actual file and not the code review page.